flusher.go

```diff
@@ -33,11 +33,8 @@ func (s *Server) Flush(ctx context.Context) {
 
 	s.Statsd.Gauge("worker.span_chan.total_elements", float64(len(s.SpanChan)), nil, 1.0)
 	s.Statsd.Gauge("worker.span_chan.total_capacity", float64(cap(s.SpanChan)), nil, 1.0)
-	s.Statsd.Gauge("gc.GCCPUFraction", float64(mem.GCCPUFraction), nil, 1.0)
 	s.Statsd.Gauge("gc.number", float64(mem.NumGC), nil, 1.0)
 	s.Statsd.Gauge("gc.pause_total_ns", float64(mem.PauseTotalNs), nil, 1.0)
-	s.Statsd.Gauge("gc.alloc_heap_bytes_total", float64(mem.TotalAlloc), nil, 1.0)
```

Comment:
Hmmm, this one was really useful when I was debugging the first iteration of the splunk span sink. I think we'll survive if we let it go, but it's a good leading indicator.

Reply:
You still get the current heap size and the pause times. It's just the "total number of bytes allocated, ever" that you're losing here.

Reply:
Oh, then good riddance!!